pkgs/by-name/wi/wivrn/package.nix

```diff
@@ -0,0 +1,147 @@
+{ config
+, lib
+, stdenv
+, fetchFromGitHub
+, fetchFromGitLab
+, applyPatches
+, autoAddDriverRunpath
+, avahi
+, boost
+, cmake
+, cudaPackages ? { }
+, cudaSupport ? config.cudaSupport
+, eigen
+, ffmpeg
+, freetype
+, git
+, glm
+, glslang
+, harfbuzz
+, libdrm
+, libGL
+, libva
+, libpulseaudio
+, libX11
+, libXrandr
+, nix-update-script
+, nlohmann_json
+, onnxruntime
+, openxr-loader
+, pipewire
+, pkg-config
+, python3
+, shaderc
+, spdlog
+, systemd
+, udev
+, vulkan-headers
+, vulkan-loader
+, vulkan-tools
+, x264
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "wivrn";
+  version = "0.17";
+
+  src = fetchFromGitHub {
+    owner = "meumeu";
+    repo = "wivrn";
+    rev = "v${finalAttrs.version}";
+    hash = "";
```

Comment:
Unsure if you forgot it or will just do this later, but the hash is missing here :)